SlidePuzzle/src/main/java/SecondApproach/Movement.java

```diff
@@ -0,0 +1,31 @@
+package SecondApproach;
+
+public class Movement {
```

Comment:
I think this class is not used anywhere else, right?

Reply:
thanks for the replies! actually my final approach is on another package (third approach) , i will clean the entire code and have this observations in mind!

Reply:
oh. I will be focusing on the Third one ...